internal/kafka/confluent_kafka.go

```diff
@@ -246,13 +246,16 @@ func consumeMessage(message *ckafka.Message, h *GroupHandler) error {
 	}
 
 	if h.Properties.PrintOffset {
-		info += fmt.Sprintf(" Partition:%d Offset:%s", message.TopicPartition.Partition, message.TopicPartition.Offset.String())
+		if len(info) > 0 {
+			info = fmt.Sprintf("%s ", info)
+		}
```

Comment:
Can we make a list of key:value pairs, and then use `strings.Join()` to combine them into a space-delimited string?

Reply:
Unit tests would be great for this, especially since we don't have integration tests here!

Reply:
added unit test!